config/tests/samples/create/harness.go

```diff
@@ -305,17 +311,40 @@ func NewHarness(ctx context.Context, t *testing.T) *Harness {
 		kccConfig.HTTPClient = &http.Client{Transport: t}
 	}
 
-	// Intercept (and log) TF requests
-	transport_tpg.DefaultHTTPClientTransformer = func(ctx context.Context, inner *http.Client) *http.Client {
-		ret := inner
-		if t := ctx.Value(httpRoundTripperKey); t != nil {
-			ret = &http.Client{Transport: t.(http.RoundTripper)}
+	if targetGCP := os.Getenv("E2E_GCP_TARGET"); targetGCP == "vcr" {
+		transport_tpg.DefaultHTTPClientTransformer = func(ctx context.Context, inner *http.Client) *http.Client {
+			ret := inner
+			if t := ctx.Value(httpRoundTripperKey); t != nil {
+				ret = &http.Client{Transport: t.(http.RoundTripper)}
+			}
+			dir := "pkg/test/resourcefixture/testdata/vcr/cassette/"
```

Comment:
Just curious: Is `cassette` a special code name? I personally don't know much about VCR and its concepts so would like to learn more context.

Reply:
`recorder` and `cassette` are both VCR concepts. Similar to the VCR recorder in the real world, this test package simulates the workflow, recording HTTP interactions instead of audio (😂), and saving them in a file, which is used to replay them later.
TBH, the folder name can be anything (`tape`, `episode`, etc., 😂), I just borrowed the name  as it's a known concept for storing recorded logs.